profiler/src/profiler/TracyTimelineItemThread.cpp

```diff
@@ -275,6 +275,14 @@ void TimelineItemThread::DrawOverlay( const ImVec2& ul, const ImVec2& dr )
     m_view.DrawThreadOverlays( *m_thread, ul, dr );
 }
 
+void TimelineItemThread::DrawExtraPopupItems()
+{
+    if( ImGui::MenuItem( ICON_FA_TIMELINE " Select in CPU timeline" ) )
```

Comment:
Is this also shown when there is no CPU timeline?

Reply:
Good point, I'll try to make it conditional